ThirdParty/nvngx_dlss_sdk/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+if (WIN32)
```

Comment:
I cleaned it a bit. Still using dynamic linking though. Turns out that to use the static path we'd need to enable the "/MT" instead of "/MD" (default) for MSVC. Which for what I see is a nasty workaround in CMake (there is an example of how this could be done in Sdl2/CmakeList.txt that is disabled by default).